### DISCUSSION TITLE
Clearing audit logs [May 15]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2113] [Discord Threads]
> 🙋‍♂️ **AK**

**Comment by o1lab** [comment]
> 👨‍🚀 **wingkwong**  
#### I think not. We don't have API for deleting audit logs  
#  
> 🙋‍♂️ **AK**  
#### Let me re-ask: How can I clear aduit logs? The best way for me is not from the db because my team who will work with my project will not be able to.  
#  
> 👨‍🚀 **wingkwong**  
#### we don't have an interface for that. You may contact your db admin to do that.   
####    
#### we'll try to provide an option to toggle audit logs. https://github.com/nocodb/nocodb/issues/2035  
#  
> 🙋‍♂️ **AK**  
#### The db admin is me.. but I will need from time to time to clear it and my team will be needed to do that and they will not understand how to do it from the db...   
#### I will follow up to see the feature, thanks.  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/974652072718917632)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
